File: Sorting/bubble_sort.js
// cmd to run : node bubble_sort.js 5 10 0 3 50 35
// 5 is size of array and rest all are elements
function bubblesort(arr){
    for (let i =0 ; i <arr.length -1; i++){
        for (let j = i; j < arr.length; j ++){
            if (arr[i] > arr[j]){
                let temp = arr[i];
                arr[i] = arr[j];
                arr[j] = temp;
            }
        }
    }
    
    console.log(arr);
}
let args = process.argv;
console.log();
let junc = 2;
let n = parseInt(args[junc]); // index 0 1 me file name and dest hoga
junc++;
let arr = [];
for (let i = 0; i<n; i++){
    arr[i] = parseInt(args[junc]);
    junc++;
}
bubblesort(arr);

File: Sorting/bubble_sort.py
def bubblesort(arr):
    for i in range(len(arr)-1):
        for j in range(i,len(arr)):
            if arr[i]> arr[j]:
                arr[i], arr[j] = arr[j], arr[i]
    return arr

print("Please enter size of the Array ")
n = int(input())
arr = [None]*n
print("Please enter elements of Array ")
for i in range(n):
    arr[i] = int(input())
print("Sorted array ")
print(bubblesort(arr))

